src/main/java/org/folio/inventory/domain/items/ItemStatusName.java

```diff
@@ -0,0 +1,52 @@
+package org.folio.inventory.domain.items;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ItemStatusName {
```

Comment:
Why is this specifically the item status’ name only, rather than the status itself?

Personally, I think this represents the status rather than the name of the status.